mantidproject/fitbenchmarking
# Fixed NIST links and redundancy.

#### Description of Work

Fixes #23
The NIST links are created only once now and are also correct.

#### Testing Instructions

1. In `example_runScripts.py`, set `run_data = "nist"`
2. Run `example_runScripts.py` in the mantidpython console.
3. Check if the links the the accuracy and runtime tables work.
4. Run `example_runScripts.py` in the mantidpython console with `run_data = "neutron"` 
5. The tables should still have the working NIST links, this is a known issue.
